src/VisualStudio/Core/Def/Implementation/ProjectSystem/DocumentProvider.cs

```diff
@@ -80,9 +85,9 @@ internal abstract partial class DocumentProvider : ForegroundThreadAffinitizedOb
 
             int installed;
             Marshal.ThrowExceptionForHR(shell.IsPackageInstalled(Guids.RoslynPackageId, out installed));
-            IsRoslynPackageInstalled = installed != 0;
+            _isRoslynPackageInstalled = installed != 0;
```

Comment:
This is dead code.  `IsRoslynPackageInstalled` is never referenced and can just be deleted.